server/database/init.sql

```diff
@@ -84,7 +84,7 @@ VALUES
 INSERT INTO
     devices (device_id, owner)
 SELECT
-    'bob_device', id
```

Comment:
Mener at deviceId starter fra 1, hvor 1 betyder det er primary device for brugeren